setup.cfg

```diff
@@ -0,0 +1,51 @@
+[bdist_wheel]
+universal = 1
+
+[metadata]
+name = ansible-bender
+url = https://github.com/TomasTomecek/ansible-bender
+description = A tool which builds container images using Ansible playbooks
+long_description = file: README.md
+long_description_content_type = text/markdown
+author = Tomas Tomecek
+author_email = tomas@tomecek.net
+license = MIT
+license_file = LICENSE
+classifiers =
+    Development Status :: 4 - Beta
+    Environment :: Console
+    Intended Audience :: Developers
+    License :: OSI Approved :: MIT License
+    Operating System :: POSIX :: Linux
+    Programming Language :: Python
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
+    Topic :: Software Development
+    Topic :: Utilities
+keywords =
+    ansible
+    containers
+    linux
+    buildah
+
+
+[options]
+python_requires = >=3.6
+packages = find:
+include_package_data = True
+use_scm_version = True
+
+setup_requires =
+    setuptools_scm
```

Comment:
```suggestion
    setuptools_scm
    setuptools_scm_git_archive
```

Reply:
so it would generate git archives using the setuptools_scm; that's pretty neat

Reply:
Oh, I forgot to mention that it requires extra file in Git tree...

Reply:
Remark: setuptools_scm doesn't generate git-archives. `git archive` command does or GitHub when users hit download on commit or tag.